src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,28 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    private List<Bulldozer> list;
+
+    {
```

Comment:
never use init blocks - it's a bad practice.
In this task better create a new list in the method.